app/presenters/taxon_presenter.rb

```diff
@@ -1,81 +1,101 @@
 class TaxonPresenter
-  GRID = :grid
-  ACCORDION = :accordion
-  LEAF = :leaf
-
   attr_reader :taxon
   delegate(
     :content_id,
     :title,
     :description,
     :base_path,
     :tagged_content,
-    :grandchildren?,
     :child_taxons,
     :live_taxon?,
-    :most_popular_content,
-    :can_subscribe?,
+    :section_content,
+    :organisations,
     to: :taxon
   )
 
+  # These should remain in order as the sequence of sections displayed on the
+  # page is important.
+  SUPERGROUPS = %w(services
+                   guidance_and_regulation
+                   news_and_communications
+                   policy_and_engagement
+                   transparency).freeze
+
   def initialize(taxon)
     @taxon = taxon
   end
 
-  def rendering_type
-    return GRID if taxon.grandchildren?
+  def sections
+    SUPERGROUPS.map do |supergroup|
+      {
+        show_section: show_section?(supergroup),
+        title: section_title(supergroup),
+        documents: section_document_list(supergroup),
+        see_more_link: section_finder_link(supergroup)
+      }
+    end
+  end
 
-    taxon.children? ? ACCORDION : LEAF
+  def section_title(supergroup)
+    supergroup.humanize
   end
 
-  def renders_as_accordion?
-    rendering_type == ACCORDION
+  def section_document_list(supergroup)
+    section_content(supergroup).each.map do |document|
```

Comment:
I think we're conflating a few different concerns here, not necessary for this PR but I'd like to see a distinct class or module handling how supergroups generate their title and content. Perhaps a bunch of subclasses or something a bit more encapsulating of the supergroup specific logic. Seems to be spread across a few different classes here.